README.md

```diff
@@ -22,13 +22,84 @@ mn create-app life.qbic.sampletracking --features=groovy --build maven
 ```
 
 ## Data model
-The data model that holds sample tracking information is denfined by attributes and relations shown in the following ER diagram.
+The data model that holds sample tracking information is defined by attributes and relations shown in the following ER diagram.
 
 ![er-diagram](models/sample-tracking-er.svg)
 
 ## API design
 The remote [RESTful API](https://app.swaggerhub.com/apis-docs/qbic/sample-tracking) is created with [swagger.io](https://swagger.io/).
 
+## Authentication
+
+The roles and user tokens must be provided in a file following the YAML format specification.
+An exemplary entry looks like this:
+
+```
+servicereader:
+    token: 123!
+    roles:
+        - READER
+    servicewriter:
+        token: 123456!
+        roles:
+            - READER
+            - WRITER
+```
+
+## Input Formats
+The accepted input formats are listed in the following
+### For samples
```

Comment:
In my opinion it's not necessary to have a subdivision into location and sample API since there are only 6 API calls overall in the service.  
```suggestion
```